src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelMailsPage/SavePersonnelError/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, useMemo } from 'react';
+import { ContactMailCollection } from '../ManagePersonnelMailContext';
+import useSavePersonnelError from './styles';
+
+type SavePersonnelErrorProps = {
+    contactMailForm: ContactMailCollection;
+};
+
+const SavePersonnelError: FC<SavePersonnelErrorProps> = ({ contactMailForm }) => {
+    const inputErrors = useMemo(
+        () => contactMailForm.filter((formItem) => !!formItem.inputError),
+        [contactMailForm]
+    );
+    const styles = useSavePersonnelError();
+    return (
+        <div className={styles.container}>
+            {inputErrors.map((inputError) => (
```

Comment:
Will this produce the 'children does not have key prop' react warning/error?

Reply:
Better safe than sorry, added the key